Makefile

```diff
@@ -0,0 +1,31 @@
+docker-build:
+	docker-compose build
+
+docker-stop:
+	docker-compose stop
+
+initial-setup:
+	docker-compose run api rake db:setup
+	docker-compose run api rake db:migrate
+
+update-db:
+	docker-compose run api rake db:migrate
+
+setup-local-db:
+	docker run  -d -v pg-data:/var/lib/postgresql/data -p 5432:5432  --name postgres  postgres:10.5
+	docker-compose run api rake db:setup db:migrate
+
+destroy-local-db:
+	@docker container stop postgres || true
+	@docker container rm postgres || true
+
+start-local-server: destroy-local-db setup-local-db
+	rails s
```

Comment:
Missing `bundle exec` here?

Reply:
i didn't need bundle exec here

Reply:
```
$ make start-local-server
[...]
rails s
Rails is not currently installed on this system. To get the latest version, simply type:

    $ sudo gem install rails

You can then rerun your "rails" command.
```

I'll try switching to `rbenv` and see if it goes away.

Reply:
Nope, switching to rbenv does not fix it.

But adding `./bin` to my `$PATH` does. Perhaps you already set that, or you have installed something that does it automatically?

Reply:
`gem install rails` to install the gem into my rbenv-managed ruby (ie. outside of the bundle) made this command work (I think because `~/.rbenv/shims/rails` appeared), so maybe that's why it was working on your machine?

I favour changing this to be './bin/rails s' as it doesn't seem ideal to require us to install gems directly into the ruby - we want applications to exclusively use bundler managed gems.